setup_cfg_fmt.py

```diff
@@ -316,6 +316,44 @@ def _is_ok_classifier(s: str) -> bool:
     return [s for s in classifiers if _is_ok_classifier(s)]
 
 
+def _imp_classifiers(setup_cfg: str) -> Optional[str]:
```

Comment:
looks like this function never returns `None` despite being marked `Optional` (it sometimes returns an empty string though!)